apps/src/templates/CodeWorkspace.jsx

```diff
@@ -138,6 +148,22 @@ class CodeWorkspace extends React.Component {
       usingBlocks && studioApp().enableShowBlockCount ? 'inline-block' : 'none';
   };
 
+  // The workspace alert will be displayed at the bottom of codeTextbox if editCode is
+  // assigned true (implies Droplet, not Blockly). Otherwise, it is displayed at the bottom
+  // of the CodeWorkspace
+  renderWorkspaceAlert() {
+    return (
+      <WorkspaceAlert
+        type="error"
+        onClose={this.props.displayWorkspaceAlertOff}
+        isBlockly={false}
+        displayBottom={true}
```

Comment:
Should 'type' and 'displayBottom' also come from Redux? i could see us wanting to configure these down the line.

Reply:
Yes, I agree! That's a good point. Will add keys.

Reply:
Discussed in Slack and will change keys to workspaceAlertType and workspaceAlertDisplayButton and add to projectRedux.js.